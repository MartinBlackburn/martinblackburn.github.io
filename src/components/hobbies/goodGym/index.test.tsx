// libraries
import { render } from "@testing-library/react";

// component under test
import GoodGymText from "./";

describe("GoodGymText component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<GoodGymText />);

        expect(asFragment()).toMatchSnapshot();
    });
});
