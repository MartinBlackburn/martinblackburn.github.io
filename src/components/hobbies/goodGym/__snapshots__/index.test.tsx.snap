// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoodGymText component renders correctly 1`] = `
<DocumentFragment>
  <p>
    I am a member of the 
    <a
      href="https://www.goodgym.org"
    >
      Good Gym
    </a>
    . Who are a group of runners who get together to help out in the community. Projects can range from tidying up public spaces, to running (literally) errands for people unable to do so themselves.
  </p>
  <p>
    While being with Good Gym I've currently run about 600km and done 118 good deeds.
  </p>
</DocumentFragment>
`;
